### ISSUE TITLE
Unbound type when using `from ... import`

### ISSUE THREAD

**Issue by NateScarlet** [microsoft/pyright#341]
**Describe the bug**
For some library, type imported with `from ... import` always unbound.

**To Reproduce**
```python
import django.core
from django import core
from django.core import exceptions
from django.core.exceptions import ValidationError

from collections import OrderedDict


def work_1():
    raise django.core.exceptions.ValidationError({})


def work_2():
    OrderedDict()


def not_work_1():
    raise ValidationError({})


def not_work_2():
    raise exceptions.ValidationError({})


def not_work_3():
    raise core.exceptions.ValidationError({})
```

**Expected behavior**
Got correct type when using `from ... import`.

**Screenshots or Code**
![image](https://user-images.githubusercontent.com/21083014/69024041-0c3db280-09fc-11ea-9c5d-1c069f8b38a6.png)

**VS Code extension or command-line**
VS Code extension: 1.1.0

**Comment by erictraut** [comment]
Thanks for the bug report. Yes, this is a regression that I introduced in the recent rewrite of the analysis engine. And it wasn't caught by any unit tests. I have a fix, and I'll publish a new version shortly.

**Comment by erictraut** [comment]
This is now fixed in v1.1.1, which I just published.
